### ISSUE TITLE
Unable to Clock Out

### ISSUE THREAD

**Issue by ndjo7189** [Shaun-Regenbaum/OneUSGAutomaticClock#14]
I've tested the program to clock for .15 hour (9-10 minutes), the program runs for 15 minutes then outputs the following:

Failed, Unable to Clock Out.
NOTICE: Please Manually Clock Out To Avoid Issues
If this error continues, please raise an issue on Github

**Comment by Shaun-Regenbaum** [comment]
Hey thanks for raising an issue. This is a known problem, but I haven't fixed because I don't know what the minimum amount of time should be. Right now it is 15 minute periods, but what do you think it should be?

Also, did the program actually not clock out, or did it just say it didn't (which is another issue I have seen)?

**Comment by kendallm** [comment]
My guess is the later, it actually clocked you out. There is currently a bug in the clock out process. It will clock you out, but say it did not for some reason.

I would just double check the webdriver window to verify that you clocked out successfully. I can take a look into the bug later today.

**Comment by Shaun-Regenbaum** [comment]
Yah, that is something I have noticed as well, I am a bit swamped today, but I will also try to take a look as well. 
I'm sure its just that I was not checking the correct element when it comes to confirming the clock out. Or perhaps we should check multiple elements as a backup.

Regardless, kudos to you Kendall for being awesome!

**Comment by ndjo7189** [comment]
I think also maybe forcing the user to input a number greater than 15 minute would help with UI. I randomly tested like .1 hour (6 minutes) when I ran the updated program for the first time. The program counted up until 15 minutes, output the same error, and actually did not clock me out. However, when I tried running the program again for 1 hour, while it threw the same error, the program did clock me out then.
